packages/debugger/src/panels/variables/mimerenderer.ts

```diff
@@ -0,0 +1,58 @@
+import { MainAreaWidget } from '@jupyterlab/apputils';
+import { IRenderMimeRegistry, MimeModel } from '@jupyterlab/rendermime';
+import { PromiseDelegate } from '@lumino/coreutils';
+import { Panel } from '@lumino/widgets';
+import { IDebugger } from '../../tokens';
+
+/**
+ * Debugger variable mime type renderer
+ */
+export class VariableMimeRenderer extends MainAreaWidget<Panel> {
+  constructor(options: VariableMimeRenderer.IOptions) {
```

Comment:
Looks like there is a missing docstring for the `constructor`.

Reply:
Done thanks @jtpio